src/components/common/subscription-form/SubscriptionForm.tsx

```diff
@@ -0,0 +1,164 @@
+              />
+              {errors.lastName && (
+                <p className="absolute py-1 text-red-500">
+                  {"Last name is required"}
+                </p>
+              )}
+            </div>
+          </div>
+          <div>
+            <label htmlFor="email" className="block text-sm font-medium">
+              Email address
+            </label>
+            <div className="relative mt-2 mb-4">
+              <input
+                {...register("email", { required: true })}
+                id="email"
+                type="email"
+                autoComplete="email"
+                className="block w-full rounded-md border-0 p-2 shadow-sm ring-gray-300 ring-1"
+              />
+              {errors.email && (
+                <p className="absolute py-1 text-red-500">
+                  {"Email is required"}
+                </p>
+              )}
+            </div>
+          </div>
+          <div>
+            <label htmlFor="country" className="block text-sm font-medium">
+              Section
+            </label>
+            <div className="relative mt-2 mb-4">
+              <select
+                {...register("section", { required: true })}
+                id="section"
+                name="section"
+                className="block w-full rounded-md border-0 p-2 shadow-sm ring-gray-300 ring-1"
+              >
+                {HEADER_NAVIGATION.map(({ name }) => (
+                  <option key={name}>{name}</option>
+                ))}
+              </select>
+              {errors.section && (
+                <p className="absolute py-1 text-red-500">{"Select section"}</p>
+              )}
+            </div>
+          </div>
+        </div>
+        <fieldset className="mt-10">
+          <legend className="text-sm font-semibold">Gender</legend>
+          <div className="mt-3 space-y-1">
+            <div className="flex items-center gap-x-3">
+              <input
+                {...register("gender")}
+                id="male"
+                name="gender"
+                type="radio"
+                value="male"
+                defaultChecked
+                className="h-4 w-4 border-gray-300"
+              />
+              <label htmlFor="male" className="block text-sm font-medium">
+                Male
+              </label>
+            </div>
+            <div className="flex items-center gap-x-3">
+              <input
+                {...register("gender")}
+                id="female"
+                name="gender"
+                type="radio"
+                value="female"
+                className="h-4 w-4 border-gray-300"
+              />
+              <label htmlFor="female" className="block text-sm font-medium">
+                Female
+              </label>
+            </div>
+          </div>
+        </fieldset>
+      </div>
+      <button
+        className="text-white px-2 py-1 rounded bg-main max-w-fit self-end"
+        type="submit"
```

Comment:
do we have disables state?

Reply:
Implemented reset form state and disable button while values are not valid